lib/type.ml

```diff
@@ -223,3 +223,34 @@ let rec of_absform = function
      of_elem (TySingleton (Atom "not_implemented"))
   | F.TyPredef {line; name; args} ->
      failwith (!%"Prease report: line:%d: unexpected predef type: '%s/%d'" line name (List.length args))
+
+let rec of_erl_type = function
+  | Erl_type.Any -> TyAny
+  | None -> TyBottom
+  | Unit -> TyBottom
+  | erl_type -> TyUnion (union_of_erl_type erl_type)
+and union_of_erl_type = function
+  | Erl_type.None -> []
```

Comment:
The case `None` is an impossible pattern. Together with `Any` and `Unit`, please make it an impossible case.

Reply:
Unfortunately, it is possible to appear `None` in `Union` as seen in the following: https://github.com/dwango/fialyzer/blob/2e8a306ea4f0bde92ec9c39720089805744a3859/test/unit-test/test_plt.ml#L255-L268